File: JAVA/9.Nested_For_2/p6.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows: ");
		int row = sc.nextInt();
		int count=1,sub=0;
		for(int i=0;i<row;i++){
		
			for(int j=0;j<row;j++){
			
				System.out.print(count+"\t");
				count++;
			}
			count=((row*(i+1))-sub);
			sub++;
			System.out.println();
		}
	}
}

File: Assignments/Assignment_18/p6.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows :\n");
        scanf("%d",&row);
        for(int i=row;i>0;i--){

                for(int j=1;j<=i;j++){

                        if(row%2!=0){

                                if(j%2!=0){

                                        printf("%d\t",j);
                                }else{

                                        printf("%c\t",(64+j));
                                }
                        }else{

                                if(j%2!=0){

                                        printf("%c\t",(64+j));
                                }else{

                                        printf("%d\t",j);
                                }
                        }
                }printf("\n\n");
        }
}
